src/DebtToken.sol

```diff
@@ -125,24 +135,40 @@ abstract contract DebtToken is ERC4626 {
                             ACCOUNTING LOGIC
     //////////////////////////////////////////////////////////////*/
 
+    /**
+     * @notice Modification of the standard ERC-4626 convertToShares implementation
+     * @dev Since debt is a liability instead an asset, roundUp and roundDown are inverted compared to the standard implementation.
```

Comment:
```suggestion
     * @dev Since debt is a liability instead of an asset, roundUp and roundDown are inverted compared to the standard implementation.
```